main.py

```diff
@@ -1,39 +1,102 @@
 #CRUD
```

Comment:
I'm treating the `main.py` as a bit of a playground right now. It should be cleaned out by either moving the code to one of the other files (model, view or controller) or deleting the code.